setup.nsi

```diff
@@ -183,9 +183,9 @@ Section "Required Files"
     File "target\${PROJECT_FINAL_NAME}.exe"
     File "target\${PROJECT_FINAL_NAME}-uber.jar"
     File "src\main\resources\img\icons\dataloader.ico"
+    File "/oname=defaultConfig.properties" "target\config.properties"
     FileOpen $9 "${PROJECT_FINAL_NAME}.l4j.ini" w
     ;Java Args here
-    FileWrite $9 "-Dappdata.dir=$\"$APPDATA$\"$\r$\n"
```

Comment:
Same as Mac profile above, do not set where `config.properties` will stored during packaging time.